BioToolsModules/SeqAnalyser.py

```diff
@@ -0,0 +1,17 @@
+def count_gc(seq:str) -> int:
+    "Function calculates CG content of a sequence (string) in percentage"
+    gc_counts = 0
+    for nuc in seq:
+        if nuc == 'C':
+            gc_counts += 1
+        elif nuc == 'G':
+            gc_counts += 1
+    return gc_counts * 100 / float(len(seq))
+
+
+def mean_quality(quality:str) -> int:
+    "Function translate ASCII quality scores (string) of sequence to num and returns mean score (from 0 to 100)"
+    quality_num = list()
+    for i in quality:
```

Comment:
```suggestion
    for q in quality:
```
хотя бы так

`i` -- уместна только для индексов
-0.5